mgmtsystem_nonconformity/mgmtsystem_nonconformity.xml

```diff
@@ -86,6 +87,7 @@
                     <group>
                         <group>
                         	<field name="ref" attrs="{'readonly':[('state','not in',['draft'])]}"/>
+                            <field name="name" attrs="{'readonly':[('state','not in',['draft'])]}"/>
```

Comment:
Hey @dreispt , do you mean always writable? The attrs acts only on readonly here.
I am confused sorry :(